tests/unit/test_spatial_methods/test_finite_volume/test_grad_div_shapes.py

```diff
@@ -74,67 +73,6 @@ def test_grad_div_shapes_Dirichlet_bcs(self):
             atol=1e-6,
         )
 
-    def test_cylindrical_grad_div_shapes_Dirichlet_bcs(self):
```

Comment:
this test is not related to the main change in this PR, it should not be removed